src/Bridge/Doctrine/Orm/Util/QueryJoinParser.php

```diff
@@ -100,35 +100,15 @@ public static function getClassMetadataFromJoinAlias(string $alias, QueryBuilder
      */
     public static function getJoinRelationship(Join $join): string
```

Comment:
I'll update the code to use those inner functions directly in the codebase in a future PR as they add no real benefits now

Reply:
cool ok